### ISSUE TITLE
spack install elfutils fails

### ISSUE THREAD

**Issue by nicksan2c** [LLNL/spack#3443]
`spack install elfutils` fails as follows

```
$ spack install elfutils
==> Installing elfutils
==> autoconf is already installed in /home/ncs/git/spack/opt/spack/linux-centos7-x86_64/gcc-4.8.5/autoconf-2.69-6mqr3qfmfzcbc6pivnzglbnkw2zyz7wd
==> libtool is already installed in /home/ncs/git/spack/opt/spack/linux-centos7-x86_64/gcc-4.8.5/libtool-2.4.6-bcbiddkmbu4gy7wx6ftntmxqe3c5br3z
==> automake is already installed in /home/ncs/git/spack/opt/spack/linux-centos7-x86_64/gcc-4.8.5/automake-1.15-j3eg5pnh22eahao7nhcpbempb6jlyydo
==> Trying to clone git repository:
  git://git.fedorahosted.org/git/elfutils.git
  at tag elfutils-0.163
Cloning into 'elfutils'...
fatal: unable to connect to git.fedorahosted.org:
git.fedorahosted.org[0: 140.211.169.199]: errno=Connection refused

Cloning into 'elfutils'...
fatal: unable to connect to git.fedorahosted.org:
git.fedorahosted.org[0: 140.211.169.199]: errno=Connection refused

==> Fetching from [git] git://git.fedorahosted.org/git/elfutils.git failed.
...
...
```

elfutils was migrated as of this commit Sat, 24 Dec 2016 21:31:41 +0000 (22:31 +0100):

https://sourceware.org/git/?p=elfutils.git;a=commit;h=1700fd25e6caf26663af2bd994d1d99fab9df59f

fedorahosted.org was retired on March 1st, 2017

https://fedoraproject.org/wiki/Infrastructure/Fedorahosted-retirement

Docs need to be updated @ https://spack.readthedocs.io/en/latest/package_list.html#elfutils

Here is the patch for relevant script

`spack/var/spack/repos/builtin/packages/elfutils/package.py
`
```
36c36
<     homepage = "https://fedorahosted.org/elfutils/"
---
>     homepage = "https://sourceware.org/git/elfutils/"
43c43
<             git='git://git.fedorahosted.org/git/elfutils.git',
---
>             git='git://sourceware.org/git/elfutils.git',

```

**Comment by nicksan2c** [comment]
Also, as of this writing, the following versions exists

elfutils-0.163
elfutils-0.164
elfutils-0.165
elfutils-0.166
elfutils-0.167
elfutils-0.168

**Comment by adamjstewart** [comment]
Feel free to contribute a PR. Otherwise someone will get to it eventually.

**Comment by tgamblin** [comment]
Fixed by #3505
